src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
```

Comment:
You should provide a fallback font-family in case the main one doesn't load. This helps to maintain the layout and readability of your site even if there are issues with the primary font.